client/login/magic-login/index.jsx

```diff
@@ -93,7 +93,7 @@ class MagicLogin extends React.Component {
 				'magic-login': true,
 				'magic-login__request_link': ! magicLoginView,
 			} }>
-				<PageViewTracker path="/login" title="Login" />
+				<PageViewTracker path="/log-in/link" title="Login" />
```

Comment:
That's a good question. As far as I know it's only used for Google Analytics. I just had a look to the [naming conventions](https://github.com/Automattic/wp-calypso/blob/master/client/lib/analytics/README.md#google-analytics-naming-conventions) and it seems we should probably use a different title for each page. I don't have access to our Google Analytics (we should post a systems request for that) but I think the following should work:

```
Login
Login > Two-Step Authentication > Authenticator
Login > Two-Step Authentication > Backup
Login > Two-Step Authentication > Push
Login > Two-Step Authentication > Sms
Login > Link
...
```